RunningOrder/Sprint/Views/SprintList.swift

```diff
@@ -17,6 +17,7 @@ extension SprintList {
         @EnvironmentObject var storyManager: StoryManager
         @ObservedObject var logic: Logic
         let space: Space
+        @State private var deletingSprint: Sprint?
```

Comment:
deletedSprint

Reply:
It is not the deleted sprint, it is the "currently deleting sprint, but not yet as we don't have the confirmation yet" xD do you have a better name ? deleted seems confusing for me as it is not yet deleted

Reply:
toBeDeletedSprint :p

Reply:
merged in the previous MR